distribution/docker/README.md

```diff
@@ -25,10 +25,6 @@ From the root of the repo, run `docker build -t apache/druid:tag -f distribution
 
 Edit `environment` to suite. Run `docker-compose -f distribution/docker/docker-compose.yml up`
 
-## Java 11
```

Comment:
https://github.com/apache/druid/blob/a27f4f57407d853d980a526636cef19003ab9d96/distribution/docker/Dockerfile#L20

The dockfile use JDK8 as the default one. So I think we still need this section to describe how to switch to JDK11 by passing `JDK_VERSION` to the `docker build` command